index.html

```diff
@@ -123,61 +123,66 @@
     </div>
 </div>
 
-<script
-src="https://code.jquery.com/jquery-3.2.1.min.js"
-integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4="
-crossorigin="anonymous"></script>
-<script src="jquery-3.2.1.min.js"></script>
 <script src="https://cdnjs.cloudflare.com/ajax/libs/crypto-js/3.1.9-1/crypto-js.min.js"></script>
-<script src="https://cdnjs.cloudflare.com/ajax/libs/jsrender/0.9.84/jsrender.js"></script>
 
 <script>
-    $(function () {
-
-        var renderTemplate = function (data) {
-
-            $.when($.get('password_template.html'))
-                .done(function (tmplData) {
-                    $.templates({tmpl: tmplData});
+    // https://github.com/sindresorhus/pupa
+    var pupaMini = function (tpl, data) {
```

Comment:
The function is not so clear outside of the context of pupa, could you use a more descriptive name for the variable name ? :)